libethereum/BlockChainSync.cpp

```diff
@@ -298,6 +299,12 @@ void BlockChainSync::onPeerNewBlock(std::shared_ptr<EthereumPeer> _peer, RLP con
 		case ImportResult::FutureTimeUnknown:
 		case ImportResult::UnknownParent:
 		{
+			if (++_peer->m_unknownNewBlocks > c_maxPeerUknownNewBlocks)
```

Comment:
anyone else to who can't immediately tell whether `++foo->` binds first left or right? or is it just me?